src/app/pages/HomePage/HomePage.tsx

```diff
@@ -2,7 +2,12 @@
 /* @jsxFrag React.Fragment */
 import { jsx } from '@emotion/react';
 
-const HomePage = () => {
-  return <div>Hi, I am a Home Page component!</div>;
+const HomePage = ({ pageData }) => {
```

Comment:
We're going to need to use TS to define the props here (or do we add this to the models directory somewhere, so that we can use it both here and in`home/getInitialData`
```suggestion
type HomePageProps = {
  pageData: {
    id: string;
    title: string;
  }
};

const HomePage = ({ pageData }) => {
```

Reply:
Suggest we could do this in the next stage of work when we come to actually return the `pageData`?

Reply:
Not having a definition for HomePage props is causing the lint build to fail, so we need to fix in order to be able to merge.